contents/docs/api/events.md

```diff
@@ -4,10 +4,11 @@ sidebar: Docs
 showTitle: true
 ---
 
+<span class='note-block'>For instructions on how to authenticate to use this endpoint, see [API Overview](/docs/api/api).</span><br />
```

Comment:
`/api/api/` looks a bit weird. Could the second `api` maybe be replaced with something else?

Reply:
Hmm I'll give it a think

Reply:
Called it "overview": /api/overview